src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -643,9 +643,14 @@ static inline int _xccdf_policy_rule_generate_fix_ansible(const char *template,
 			}
 		}
 		else {
-			char *remediation_part = malloc((ovector[0] + 1) * sizeof(char));
-			memcpy(remediation_part, &fix_text[start_offset], ovector[0]);
```

Comment:
I believe the `malloc`ed  and copyed size should be `ovector[0] - start_offset +1`

Reply:
I agree, kudos for figuring out the true intention of the code!